README.md

```diff
@@ -173,6 +173,19 @@ Possible values of this variable:
 * `{state: absent}`: the ansible host name is unset in the insights-client config file and Host Based Inventory (HBI) is updated to use the system host name.
 * any other string value: the ansible host name is changed in Host Based Inventory (HBI).
 
+```yaml
+    rhc_insights:
+      display_name: "Example Host"
+```
+
+Configures the display name with a custom value for the system record in Host Based Inventory (HBI). This display name is only used to identify the host in the Inventory.
+It is `null` by default and will use the system host name if not specified.
+
+Possible values of this variable:
+
+* `null` or an empty string: the display name is not changed
+* any other string value: the display name is changed in Host Based Inventory (HBI).
```

Comment:
Do we need a `NOTE: Once set, the display name cannot be removed or reverted back to the original value.  You can only set a new value.`
?

Reply:
Strictly speaking, there is always a display name set: in particular, the default one that `insights-client` sets (if not specified otherwise) is the hostname.

I would either write something slightly more verbose along the lines of

> Note: `insights-client` sets the default display name on registration as the hostname of the system, in case it is not explicitly configured. When changing the display name manually using this variable, it is not possible to "revert it" automatically to be the hostname, and thus it must be done manually if wanted.

... or nothing at all.

Reply:
Is is true that it’s the `insights-client` that sets the display name to the hostname, or is it sent empty and set automatically in the cloud?

Reply:
Answer: It’s not the Client that sets the default display name. If not set, it doesn’t send it at all:

- no _display_name_ field in the canonical facts JSON;
- no _display_name_ file in the archive.

That means that the system gets its display name when processed in the Platform, not on the machine.